### PR TITLE
SDK-807 extend task set

### PR DESCRIPTION
**Pull request by sergeyspryker**
https://spryker.atlassian.net/browse/SDK-807

### CODE REVIEW

File: src/Infrastructure/Service/TaskManager.php [hunk 1]
```diff
@@ -55,6 +64,10 @@ public function initialize(array $tasks): array
                 continue;
             }
 
+            if ($task instanceof TaskSetInterface) {
```
Comment (vol4onok) [inline src/Infrastructure/Service/TaskManager.php:67]: it's not correct way to extract taskset. you did it from the persistence repository. This condition will be false every time. you need to move it to the task saver. When we save tasks from yaml repository.

**Comment by vol4onok** [comment]
it's blocked by https://github.com/spryker-sdk/sdk/pull/159

**Review by vol4onok — APPROVED** [review]
But needs to wait qa automation epic and move all action for task set to taskSetConvertor that we already have

File: composer.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
         "spryker-sdk/acp": "^0.2.0",
         "spryker-sdk/async-api": "^0.2.3",
         "spryker-sdk/composer-replace": "dev-master",
-        "spryker-sdk/sdk-contracts": "^0.2.0",
+        "spryker-sdk/sdk-contracts": "dev-bugfix/sdk-807-task-set-stop-on-error-and-tags as 0.2.0",
```
Comment (pavelmaksimov25) [inline composer.json:35]: Please update the contracts branch and update the alias version as 0.3.0

**Comment by sergeyspryker** [comment]
Duplication https://spryker.atlassian.net/browse/SDK-899

**Pull request closed without merging**